content/public/browser/brave_render_frame_host.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "content/public/browser/render_frame_host.h"
+
+#include "brave/content/browser/cosmetic_filters_communication_impl.h"
+
+namespace content {
+
+RenderFrameHost::RenderFrameHost() {
+  cosmetic_filters_communication_impl_ = nullptr;
```

Comment:
just wondering, cannot we put it to a RenderFrameObserver? To avoid patching?

Reply:
I don't remember exactly what was there, but I've tried that way and there were some complications. I'm going to create a follow up to loo at that.

Reply:
https://github.com/brave/brave-browser/issues/13523

Reply:
I think we should be using `ContentBrowserClient::ExposeInterfacesToRenderer`/`ContentRendererClient::ExposeInterfacesToBrowser`